common/src/model_registration/register.py

```diff
@@ -90,21 +95,18 @@ def main(args):
     model_description = args.model_description
     registry_name = args.registry_name
     model_path = args.model_path
-    model_info_path = args.model_info_path
     registration_details = args.registration_details
+    tags, properties = {}, {}
 
     ml_client = get_ml_client(registry_name)
 
     model_info = {}
-    if model_info_path:
-        with open(model_info_path) as f:
+    if args.download_details:
+        with open(args.download_details) as f:
             model_info = json.load(f)
 
-    model_name = model_info.get("name", model_name)
-    model_type = model_info.get("type", model_type)
-    model_description = model_info.get("description", model_description)
-    tags = model_info.get("tags", {})
-    properties = model_info.get("properties", {})
+    model_name = model_name or model_info.get("model_name").replace("/", "-")
```

Comment:
`model_info.get("model_name").replace()` will fail if model_name is not provided and model_name does not exist in model_info. Can you please reach an exception when this happens and do necessary alterations to model_name post that?

Reply:
Done and exception will get raised if it will not get name from any of source(i.e model name parameter and model info)